tools/buildsys/src/args.rs

```diff
@@ -212,11 +235,14 @@ const VARIANT: u8 = BuildFlags::Variant as u8;
 #[test]
 fn build_type_includes_test() {
     // true
+    assert!(BuildFlags::Repack.includes(8));
```

Comment:
This isn't a useful test - it is effectively just asserting that the enum variant still has the value `0b0001000`, which would not survive a refactor.

Reply:
This was to avoid `REPACK` being flagged as dead code, but I can add a `clippy` exception.